firebase-test/src/class-rules.test.ts

```diff
@@ -6,6 +6,17 @@ import {
   tearDownTests, thisClass
 } from "./setup-rules-tests";
 
+// duplicated from firestore-schema.ts since we can't reach outside our folder
```

Comment:
Suggestion: consider also adding a comment to the firestore-schema.ts file to point to the duplication here; to promote keeping them in sync.?